core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -755,8 +782,14 @@ public PersistenceState getPersistenceState(long fileId) throws FileDoesNotExist
       throws AccessControlException, FileDoesNotExistException, InvalidPathException {
     Metrics.GET_FILE_INFO_OPS.inc();
     try (JournalContext journalContext = createJournalContext();
-        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ)) {
-      mPermissionChecker.checkPermission(Mode.Bits.READ, inodePath);
+        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ);
+        MasterAuditContext auditContext = createAuditContext("listStatus", path, null, null)) {
```

Comment:
There is a strict ordering of these resources, so that should be commented on in the top of this file (similar to how the journal context is commented).